packages/component-base/src/a11y-announcer.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright (c) 2022 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+
+const region = document.createElement('div');
+
+region.style.position = 'fixed';
+region.style.clip = 'rect(0px, 0px, 0px, 0px)';
+region.setAttribute('aria-live', 'polite');
```

Comment:
Can this be removed, since it always gets set in `announce()`?

Reply:
We mainly use this for testing to get reference to the live region. 
Otherwise it would require some `data-announcer` or `id` attribute.